manifests/README.md

```diff
@@ -1,32 +1,55 @@
-# Dashboard
+# Manifests
 
-The Open Data Hub Dashboard component installs a UI which 
+The Dashboard manifests run on Kustomize. There are 3 types of deployments for the Dashboard component.
 
-- Shows what's installed
-- Show's what's available for installation
-- Links to component UIs
-- Links to component documentation
+- Open Data Hub ([`./odh`](./odh))
+- Red Hat OpenShift AI
+  - RHOAI Managed ([`./rhoai/addon`](./rhoai/addon))
+  - RHOAI Self Managed ([`./rhoai/onprem`](./rhoai/onprem))
 
-For more information, visit the project [GitHub repo](https://github.com/opendatahub-io/odh-dashboard).
+## Adding/Modifying Manifests
 
-### Folders
-1. base: contains all the necessary yaml files to install the dashboard
+Rules for keeping the manifest files in a sane order:
 
-##### Installation
-Use the `kustomize` tool to process the manifest for the `oc apply` command.
+1. When adding a new type of thing, always make it have its own folder; be sure to add the reference to the parent folder's `kustomziation.yaml` (if applicable) 
+2. When adding to a preexisting folder, be sure to add it to the root `kustomization.yaml` in that folder
+3. Do not reference "a file" (has an extension) inside another folder. Reference other folders, which will pick up the `kustomization.yaml` inside that folder; those `kustomization.yaml` files should reference sibling files
+4. Folders matter -- see the README in each for more details
 
+## Installation (ODH)
+
+You can use the `kustomize` tool to process the manifest for the `oc apply` command.
+
+```markdown
+# Set the namespace in the manifest where you want to deploy the dashboard
+kustomize edit set namespace <DESTINATION NAMESPACE>
+kustomize build common | oc apply -f -
+kustomize build overrides/base | oc apply -f -
 ```
-# Parse the base manifest to deploy ODH Dashboard WITHOUT the required configs for groups
-cd manifests/base
-kustomize edit set namespace <DESTINATION NAMESPACE>   # Set the namespace in the manifest where you want to deploy the dashboard
-kustomize build . | oc apply -f -
-```
 
+## Testing Changes
+
+One way to test changes locally is to generate the full structure before your changes, and then again after your changes.
+
+Before:
+```markdown
+# Generate the files before your changes for a baseline
+kustomize build rhoai/addon > before-rhoai-addon-test.yaml
+kustomize build rhoai/onprem > before-rhoai-onprem-test.yaml
+kustomize build odh > before-odh-test.yaml
```

Comment:
We don't use `makefile` a lot, it's only scoped right now for builds and deployments, but it might be nice to have the manifest testing under a phony target there too.

Reply:
Interesting, I'll look into it.

Reply:
I'm thinking in order to do this via the makefile -- it would need to sorta do:

* be on your branch with changes
* take a snapshot of the output (after files)
* checkout "main"(?) (hopefully your local main is up to date?)
* take a snapshot of the output (before files)
* report the difference (output files)

And then I guess try to get you back on your branch again? Not sure how that works out. It's brittle and requires everyone to have the same structure of git. I'm not inclined to do this as the steps are copy-able and paste-able into the console pretty easily. 

It was an interesting thought though. We should log something @lucferbux if there is a path forward I'm not seeing here.